extensions/geohash-encode-decode/src/helpers/actions.ts

```diff
@@ -0,0 +1,8 @@
+import { shouldPopToRoot } from "./prefs";
+import { popToRoot } from "@raycast/api";
+
+export function afterActionHandler(): void {
+  if (shouldPopToRoot()) {
+    popToRoot();
```

Comment:
```suggestion
    closeMainWindow();
    popToRoot();
```
I will recommend calling `closeMainWindow()` before `popToRoot()`, since you can see a small flicker before it's closing the window